src/helpers.js

```diff
@@ -617,3 +618,12 @@ const uniqid = (function() {
 })();
 
 export {uniqid};
+
+/**
+ * Blacklist certain keys to prevent Prototype Pollution
+ * @param {any} key  - Object key to check
+ * @return {boolean} - Key is blacklisted or not
+ */
+function isPrototypePolluted(key) {
+  return /__proto__|constructor|prototype/.test(key);
```

Comment:
This is not performant at all since the regex is created each time the function is run and this fonction will be called many times since it is in a hot path. Also, on so few items I'd wager a simple succession of ifs or a set membership test would be faster than the regex. But I may be wrong and you can try and benchmark it if you want.

Reply:
What's more it is not correct since it will prevent legitimate keys such as `prototypeInterpolation`, for instance, to be written since your regex does not contain `^` nor `$`.